app/models/ability.rb

```diff
@@ -4,10 +4,13 @@ class Ability
   include CanCan::Ability
 
   def initialize(user)
-    return unless user.present?
-    if user.has_role? :admin
-      if user.has_role? :superuser
-        can [:create, :update, :destroy], User
+    can :read, PastEvent
+    if user.present?
+      if user.has_role? :admin
```

Comment:
admin's roles are subset of superuser, so i guess it might be better :admin or :superuser ?

Reply:
Done